edward/util/random_variables.py

```diff
@@ -714,6 +715,93 @@ def get_variables(x, collection=None):
   return list(output)
 
 
+def is_independent(a, b, condition=None):
+  """Assess whether a is independent of b given the random variables in
+  condition.
+
+  Implemented using the Bayes-Ball algorithm[1].
+
+  Args
+    a : RandomVariable or list of RandomVariable
+       Query node(s).
+
+    b : RandomVariable or list of RandomVariable
+       Query node(s).
+
+    condition : RandomVariable or list of RandomVariable, optional
+       Random variable(s) to condition on.
+
+  Returns
+    bool
+    True if a is independent of b given the random variables in condition.
+
+  References
```

Comment:
Can you add a bibtex entry to `docs/tex/bib.bib`? Also note the bib file is sorted chronologically by year, then sorted alphabetically by author's last name within a year.